db/dcrpg/go.mod

```diff
@@ -25,7 +25,7 @@ require (
 	github.com/decred/dcrdata/blockdata/v5 v5.0.1
 	github.com/decred/dcrdata/db/cache/v3 v3.0.1
 	github.com/decred/dcrdata/db/dbtypes/v2 v2.2.1
-	github.com/decred/dcrdata/explorer/types/v2 v2.1.1
+	github.com/decred/dcrdata/explorer/types/v2 v2.1.2
```

Comment:
There is no v2.1.2 yet, just 2.1.1.  The replace will use the current version though.
git tag -l | grep explorer/types

Reply:
Done, with the replaces in place it should be all good.